test/runtime/values.spec.js

```diff
@@ -7,6 +7,105 @@ var JuttleMoment = require('../../lib/moment').JuttleMoment;
 var Filter = require('../../lib/runtime/filter');
 
 describe('Values tests', function () {
+    var FILTER_AST = {
+        type: 'ExpressionFilterTerm',
+        expression: {
+            type: 'BinaryExpression',
+            operator: '<',
+            left: { type: 'Variable', name: 'a' },
+            right: { type: 'NumericLiteral', value: 5 }
+        }
+    };
+
+    var AST_TESTCASES = [
```

Comment:
This is probably a matter of taste, but I think its usually better to keep the test cases close to the actual tests. If you prefer constants, would you mind moving the other tests, too, please?